zerver/views/user_settings.py

```diff
@@ -233,7 +233,7 @@ def json_change_settings(
 
         do_change_password(user_profile, new_password)
         # In Django 1.10, password changes invalidates sessions, see
-        # https://docs.djangoproject.com/en/1.10/topics/auth/default/#session-invalidation-on-password-change
+        # https://docs.djangoproject.com/en/3.2/topics/auth/default/#session-invalidation-on-password-change
```

Comment:
This is referring specifically to a change in 1.10. We probably don’t care about the exact history anymore, but if not, the reference to 1.10 above should be removed too.

Reply:
Yeah, I think just "Password changes invalidate sessions, see ..." is what we want.